src/codeu/chat/common/ConversationHeader.java

```diff
@@ -48,19 +49,38 @@ public ConversationHeader read(InputStream in) throws IOException {
       );
 
     }
+    //assigning permission levels to users 
   };
 
+  public Integer permissionLevel(String title) throws IOException{
+    if (title.equals("member")){
+      return 1;
+    }
+    else if (title.equals("owner")){
+      return 2;
+    }
+    else if (title.equals("creator")){
+      return 3;
+    }
+    else {
+      throw new IOException("Please enter a valid user category.");
+    }
+  } 
+
   public final Uuid id;
   public final Uuid owner;
   public final Time creation;
   public final String title;
+  public HashMap<Uuid, Integer> user_category = new HashMap<Uuid, Integer>();
```

Comment:
java variable names need to be camel case without underscores. Use `userCategory` instead.